tests/wpunit/Tribe/Tickets/Partials/V2/RSVP/ARI/Sidebar/TitleTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Tribe\Tickets\Partials\V2\RSVP\ARI\Sidebar;
+
+use Codeception\TestCase\WPTestCase;
+use Spatie\Snapshots\MatchesSnapshots;
+use tad\WP\Snapshots\WPHtmlOutputDriver;
+use Tribe\Test\PHPUnit\Traits\With_Post_Remapping;
+use Tribe\Tickets\Test\Commerce\RSVP\Ticket_Maker as RSVP_Ticket_Maker;
+
+class Title extends WPTestCase {
+
+	use MatchesSnapshots;
+	use With_Post_Remapping;
+
+	use RSVP_Ticket_Maker;
+
+	protected $partial_path = 'v2/rsvp/ari/sidebar/title';
+
+	/**
+	 * @test
+	 */
+	public function test_should_render_ar_sidebar_title() {
+		$template  = tribe( 'tickets.editor.template' );
+		$event     = $this->get_mock_event( 'events/single/1.json' );
+		$event_id  = $event->ID;
+		$ticket_id = $this->create_rsvp_ticket(
+			$event_id,
+			[
+				'meta_input' => [
+					'_capacity' => 5,
+				],
+			]
+		);
+
+		$ticket = tribe( 'tickets.rsvp' )->get_ticket( $event_id, $ticket_id );
```

Comment:
I could be wrong here, but I feel like we should not be using `tribe()` within template tests (since we're only focusing on template outputs). Maybe @lucatume has thoughts on this?